src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    this.setAnimal();
+  }
+
+  setAnimal() {
+    Animal.alive = [...Animal.alive, this];
+  }
+
+  removeAnimal() {
+    Animal.alive = Animal.alive.filter((el) => el.health > 0);
```

Comment:
The removeAnimal method is not correctly removing the specific instance of the animal that should be removed. It filters out all animals with health greater than zero, which is not the intended behavior. Consider modifying this method to remove only the specific instance of the animal whose health is zero or less.